src/common/JSHandle.ts

```diff
@@ -837,8 +837,11 @@ export class ElementHandle<
         try {
           await fs.promises.access(resolvedPath, fs.constants.R_OK);
```

Comment:
@OrKoN  Should we just remove this check in its entirety? The browser will throw itself if it really doesn't exist, right?

Reply:
Do we have a history for why it was introduced? Could we distinguish if the browser is running locally or not and validate accordingly?

Reply:
I don't think validating here is useful, but I'll investigate higher in the stack. Also, rather than validating existence, how about asserting that an absolute path must be given?

Reply:
the api doc says that we would resolve relative paths too. We could require an absolute path if we know that the browser is remote though.